sdks/java/io/debezium/src/main/java/org/apache/beam/io/debezium/KafkaSourceConsumerFn.java

```diff
@@ -94,7 +95,7 @@ public KafkaSourceConsumerFn(
       Class<?> connectorClass, SourceRecordMapper<T> fn, long minutesToRun) {
```

Comment:
I think we made a mistake making `KafkaSourceconsumerFn` public. Can you remove the public keyword here if possible?

Reply:
Yes, done